api/database/models.py

```diff
@@ -0,0 +1,96 @@
+from datetime import datetime, timezone
+from sqlalchemy import Column, String, Integer, Text, Boolean, DateTime
+from enum import Enum
+from api import db
+
+class Skill(db.Model):
+    __tablename__ = 'skills'
+
+    id = Column(Integer, primary_key=True)
+    name = Column(String(80), nullable=False)
+
+    created_at = Column(DateTime(timezone=True), nullable=False, default=datetime.now())
+
+    # Many to Many Relationship: nothing needed here.
+    applicants = db.relationship('Applicant', secondary='applicant_skills')
+
+class Value(db.Model):
+    __tablename__ = 'values'
+
+    id = Column(Integer, primary_key=True)
+    name = Column(String(80), nullable=False)
+
+    created_at = Column(DateTime(timezone=True), nullable=False, default=datetime.now())
+
+    # Many to Many Relationship: nothing needed here.
+    applicants = db.relationship('Applicant', secondary='applicant_values')
+
+class ApplicantSkill(db.Model):
+    __tablename__ = 'applicant_skills'
+    id = db.Column(db.Integer, primary_key=True)
+    applicant_id = db.Column(db.Integer, db.ForeignKey('applicants.id'))
+    skill_id = db.Column(db.Integer, db.ForeignKey('skills.id'))
+
+    created_at = Column(DateTime(timezone=True), nullable=False, default=datetime.now())
+
+    # relationships
+    applicant = db.relationship('Applicant', backref=db.backref("applicant_skills", cascade="all, delete-orphan"))
+    skill = db.relationship('Skill', backref=db.backref("applicant_skills", cascade="all, delete-orphan"))
```

Comment:
can you share the resources you used to guide setting up these model relationships?

Reply:
Yes, I've added a link to the docs for flask-sqlalchemy to the Notes: BE resources card